### ISSUE TITLE
aws-profile fails with python error regarding timedelta method

### ISSUE THREAD

**Issue by anosulchik** [Smartling/aws-profile#4]
Newly installed aws-profile (installed on fresh EC2 instance) fails to execute with the following error:

```
Traceback (most recent call last):
  File "/usr/sbin/aws-profile", line 235, in <module>
    creds = load_creds()
  File "/usr/sbin/aws-profile", line 217, in load_creds
    if creds is None or not is_creds_valid(creds):
  File "/usr/sbin/aws-profile", line 208, in is_creds_valid
    return now + datetime.timedelta(minutes = 5) <= end_time
AttributeError: type object 'datetime.datetime' has no attribute 'timedelta'`
```

It's related to this pull request (it works fine without line that was added in PR below).

https://github.com/Smartling/aws-profile/pull/3

Please advise. Thank you.

**Comment by anosulchik** [comment]
@enisher

**Comment by maescomua** [comment]
getting the same

```
python --version
Python 2.7.12
```

**Comment by maescomua** [comment]
fixed in 1fd548d
